qgreenland/cli/fetch.py

```diff
@@ -26,6 +20,12 @@
 @click.argument('pattern')
 def fetch(pattern, dry_run, workers) -> None:
     """Fetch assets for datasets matching PATTERN."""
+    from qgreenland.util.config.config import (
+        get_config,
+        init_config,
+    )
+    from qgreenland.util.luigi import fetch_tasks_from_dataset
```

Comment:
The linked issue indicates why we did this but it might be nice to include a comment in the code for these imports? I could see myself forgetting.

Reply:
Me too. I'll add comments!